judgels-backends/judgels-commons/judgels-service-core/src/main/java/judgels/service/ServiceUtils.java

```diff
@@ -60,6 +64,38 @@ public static Response buildDownloadResponse(String fileUrl) {
         }
     }
 
+    public static Response buildImageResponseFromText(String text, Date lastModifiedStream) {
+        int fontSize = 14;
+        int margin = 30;
+        int charWidth = 8;
+        int charHeight = 17;
+
+        String[] textList = text.split("\n");
```

Comment:
Can we verify if this works correctly if the submitted file has windows-style line endings (`\r\n`)?